directory/src/App.js

```diff
@@ -67,12 +67,17 @@ class SearchBar extends PureComponent {
   timer = null;
 
   state = {
+    selectedFamilies: Object.keys(IconFamilies), // initially all families are selected
```

Comment:
If no families should be suggested:

```suggestion
    selectedFamilies: [],
```

```suggestion
    selectedFamilies: Object.keys(IconFamilies).filter(family => family !== 'FontAwesome5Brands'),
```

Anyway, the comment is not really necessary 🙂

Reply:
I change the code to have no family selected at the beginning.